chaos.py

```diff
@@ -1,34 +1,22 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-import time
 import os
+from os.path import dirname, abspath, join
+
+import time
 import sys
 import logging
-import threading
-import http.server
-import random
 import subprocess
 import settings
-import patch
 import schedule
 
-from os.path import dirname, abspath, join
-
 import cron
 import github_api as gh
-import github_api.prs
-import github_api.voting
-import github_api.repos
-import github_api.comments
-
-# Has a sideeffect of creating private key if one doesn't exist already
-import encryption
-
-from github_api import exceptions as gh_exc
 
 
 def main():
+    """main entry"""
```

Comment:
If your linter is telling you to add vacuous documentation, you need to fix your linter. May I suggest flake8?